docs/run/run-jobs-batch.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: Run jobs in a batch
+description: How to run jobs in batch mode. 
+
+---
+
+# Run jobs in a batch
+
+Batch mode can shorten processing time if all jobs can be provided at the outset. If you want to submit iterative jobs, use [sessions](sessions) instead.  Using batch mode has these benefits:
+
+-  The jobs' classical computation, such as compilation, is run in parallel. Thus, running multiple jobs in a batch is significantly faster than running them serially. 
+-  There is no delay between job, which can help avoid drift.
+
+<Admonition type="note">
+    When batching, jobs are not guaranteed to run in the order they are submitted.
+</Admonition>
+
+![Batch execution diagram.](/images/run/batch.png 'Figure 3: Batch execution')
+
+The following example shows how you can divide up a long list of circuits into multiple jobs and run them as a batch to take advantage of the parallel processing.
+
+```python
+jobs = []
+with Batch(backend) as batch:
+    estimator = Estimator(batch)
+    # calls within this context are part of the batch.
+    for obs_set in observable_sets:
+        jobs.append(estimator.run(circuits, observables=obs_set))
```

Comment:
Do we want a full example of using `Batch` (e.g. including import statements)? Since unlike `Session`, there isn't a separate page shows the full code. Or can we just rely on the soon-to-come tutorials?

Reply:
A full example might be a good idea - @pandasa123 said that the tutorials that I linked to were updated to use Batch today

Reply:
I don't see Batch in the Grover's one though; and the CHSH one is slightly different and doesn't use the with Batch() format that we have in this file

Reply:
They've been updated so I just pointed to them.